package.json

```diff
@@ -100,20 +101,14 @@
     "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8"
```

Comment:
Since we're targeting Node.js 12, we can upgrade ESLint requirement to `^8`. ESLint v7.32.0 is the last version that supports Node.js 10.

Reply:
I haven't seen what we can benefit from this yet, but I am pretty sure there are some:

- `eslint-plugin-import` might use some outdated ESLint API that the newer version of ESLint has provided a better alternative
- Unblocks ESLint's peer dependency (like `eslint-utils`) from being updated.
- Reduce CI workload.

So let's do this.

Reply:
Maybe we should drop until `eslint@7`? There are still many projects using `eslint@7`: https://github.com/search?q=%22eslint%22%3A+%22%5E7+%22eslint-plugin-import%22&type=code

Reply:
I believe `^7.2.0 || ^8` is fine?

Reply:
@JounQin @g-plane Done in [`963b028` (#11)](https://github.com/un-es/eslint-plugin-i/pull/11/commits/963b028bff920a88f9ffe9e2a4494b41b6dacdeb)